docs/backend-requests/testing/postman-or-insomnia.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Testing with Postman and Insomnia
+description: Learn about generating and using a long lived JWT Token with Postman or Insomnia.
+---
+
+# Testing with Postman and Insomnia
+
+Postman and Insomnia are powerful tools used to test API's, validate their behavior, and automate testing tasks. Basic testing with these tools is similar, and this guide will help you get started with either option.
+
+## Generate Long Lived JWT Template
+
+The standard token issued by Clerk expires after 60 seconds. Clerk SDKs handle refreshing the token regularly so that the authentication state is kept up to date. Because this token expires quickly, it isn't very useful when trying to test with Postman or Insomnia.
+
+You will want to create a long lived [JWT Template](/docs/backend-requests/making/jwt-templates) to be used in Postman or Insomnia. To do so, navigate to the [JWT Templates](https://dashboard.clerk.com/last-active?path=jwt-templates) page in the Clerk Dashboard. Click on the **New template** button and select the **Blank** template.
+
+<Images
+  src="/docs/images/testing/blank-jwt-template.webp"
+  width={3024}
+  height={1654}
+  alt="The JWT Template page in the Clerk Dashboard. The 'New template' button was clicked, in the modal that opened, the 'Blank' template is hovered over."
+/>
+
+Give your template a unique name, such as *'testing-template'*. Set the **Token Lifetime** to a value that suits your needs, or use the maximum of 315360000 seconds (10 years). If your normal token has Claims, then you can add the same Claims to your JWT Template.
```

Comment:
Security concerns should probably be added to https://clerk.com/docs/backend-requests/making/jwt-templates#creating-a-template and the linked to from this doc. For the sake of this kind of testing, the concern would be roughly like that of an API Key.

For recommended expiry time, I think most developers are using at least days if not weeks or years. Grabbing and updating tokens for this is pain and most people are looking to avoid that.

Reply:
Security concerns should probably be added to https://clerk.com/docs/backend-requests/making/jwt-templates#creating-a-template and the linked to from this doc. For the sake of this kind of testing, the concern would be roughly like that of an API Key.

For recommended expiry time, I think most developers are using at least days if not weeks or years. Grabbing and updating tokens for this is pain and most people are looking to avoid that.

Reply:
So when we say "if your normal token claims", is this referring to custom claims the user would have added?

Reply:
Updated this to "If you have added custom Claims to the normal Session token, then you should add the same Claims to your JWT Template."